.circleci/config.yml

```diff
@@ -1,35 +1,60 @@
-# SlackJanitor configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:10
+
 version: 2
 jobs:
   build:
-    docker:
-      - image: circleci/node:10
-
-    working_directory: ~/repo
-
-    branches:
-      ignore:
-        - /none
-
+    <<: *defaults
     steps:
       - checkout
-
-      # Download and cache dependencies
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-dependencies-
-
-      - run: yarn install
-
+      - run:
+          name: yarn install
+          command: yarn
       - save_cache:
+          key: v1-{{ arch }}-cache-{{ checksum "package.json"}}
           paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
-
-      # run tests!
-      - run: yarn test
+            - ./node_modules
```

Comment:
Do these run on every build or just on PRs?

Reply:
Currently, I have enabled the CI/CD to run on every commit because I am testing CI/CD now.
Once CI/CD set up is done, I'll simply change it to Pull request only in the setting.